test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -164,35 +164,82 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
             Assert.AreEqual("string", type);
 
             var searchViewModel = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
-            searchViewModel.PortViewModel = inPorts[1];
+            searchViewModel.PortViewModel = inPorts[0];
```

Comment:
I think if you make this change you should change the assertion above on line 164 so we have a clue as to what this is testing...

Reply:
But isn't that(`var[]..[]`) already being asserted in line 160?
Or are you suggesting changing their order?